tests/test_features/test_base.py

```diff
@@ -47,3 +48,57 @@ def test_subclass(self):
     #     visualizer = FeatureVisualizer()
     #     with self.assertRaises(NotImplementedError):
     #         visualizer.poof()
+    
+class DataVisualizerBaseTests(MultiFeatureVisualizer):
```

Comment:
The test suite shouldn't be a `MultiFeatureVisualizer` right? It should just be an `object`.

Reply:
Also, I think this might need to be `TestDataVisualizerBase` per [Conventions for Python test discovery](https://docs.pytest.org/en/3.0.0/goodpractices.html#test-discovery) -- the `Tests` suffix is for `unittest`, e.g. if you extended `VisualTestCase` -- but we're trying to move away from that per #318

Reply:
My bad! Will correct it.